### PR TITLE
Update Appointment classes

### PR DESCRIPTION
**Pull request by wang-h-z**
Updated Appointment classes to use Java LocalDate and LocalTime objects. This facilitates the `today` command which
will be added soon.

Edited the associated Parser classesand JUnit tests to accomodate for the update as well.

### CODE REVIEW

**Review by lrongyi — COMMENTED** [review]
Good use of Singleton objects to handle empty values for `Date`, `To`, and `From`.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java
Comment (e1121208) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:1]: well done for this

File: src/main/java/seedu/address/model/appointment/From.java
Comment (e1121208) [inline src/main/java/seedu/address/model/appointment/From.java:1]: Great job

**Review by e1121208 — COMMENTED** [review]
Overall, LGTM

**Pull request merged**